tests/results/AbsolutePositionTest.py

```diff
@@ -73,3 +73,18 @@ def test_instantiation(self):
         self.assertEqual(uut.position, 5)
         self.assertEqual(uut.line, None)
         self.assertEqual(uut.column, None)
+
+    def test_ordering(self):
+        text1 = ('A Lannister always pays his debts',)
+        text2 = ('Hello from the other side',)
+        uut1 = AbsolutePosition(text1, 4)
+        uut2 = AbsolutePosition(text1, 8)
+        uut3 = AbsolutePosition(text2, 4)
+        uut4 = AbsolutePosition(text2, 4)
+
+        self.assertTrue(uut1 <= uut2)
```

Comment:
It's too see whether `uut1 <= uut2` is also working as expected. You can override behaviour in Python for operators like `<` or `>` or `<=` (and actually we are doing that with this commit by using `@generate_ordering`).

That's the whole purpose of the test, to see whether really everything works as expected (although it seems clear). "Should work" and "does work" are two different things ;D